src/platforms/apple/troubleshooting.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Troubleshooting
+description: "Troubleshoot and resolve common issues with the Cocoa SDK."
+sidebar_order: 2000
+---
+
+## Wrong App Start Data
+
+Starting with iOS 15, the system might [pre-warm](https://developer.apple.com/documentation/uikit/app_and_environment/responding_to_the_launch_of_your_app/about_the_app_launch_sequence#3894431) your app by creating the process before the user opens it. In such cases, we can't reliably measure the app start, so we drop it as of [sentry-cocoa 7.18.0](https://github.com/getsentry/sentry-cocoa/releases/tag/7.18.0). We are working on a fix for this. Follow the [GitHub issue](https://github.com/getsentry/sentry-cocoa/issues/1897) for more details.
+
+## High Slow and Frozen Frames Rate
+
+We recommend updating to at least [7.18.1](https://github.com/getsentry/sentry-cocoa/releases/tag/7.18.1), or else the SDK might report too high a percentage of slow frames.
+
+## Falsely Reported Out Of Memory Crashes
+
+We recommend updating to at least [7.11.0](https://github.com/getsentry/sentry-cocoa/releases/tag/7.11.0), because the SDK might falsely report out-of-memory crashes when an app hangs, and the user kills it manually.
```

Comment:
```suggestion
We recommend updating to at least [7.11.0](https://github.com/getsentry/sentry-cocoa/releases/tag/7.11.0), because before this version the SDK might falsely report out-of-memory crashes when an app hangs, and the user kills it manually.
```